app/components/forms/wizard/sessions-speakers-step.js

```diff
@@ -200,11 +216,12 @@ export default Component.extend(EventWizardMixin, FormMixin, {
     onChange() {
       this.onValid(() => {});
     },
-    moveItem(item, type, direction) {
-      const index = item.get('position');
-      const otherItem = this.data.event.get(type).find(otherItem => otherItem.get('position') === (direction === 'up' ? (index - 1) : (index + 1)));
-      otherItem.set('position', index);
-      item.set('position', direction === 'up' ? (index - 1) : (index + 1));
+    moveRoom(item, direction) {
+      const idx = item.get('position');
+      const otherIdx = direction === 'up' ? (idx - 1) : (idx + 1);
+      const other = this.data.microlocations.find(item => item.get('position') === otherIdx);
+      other.set('position', idx);
+      item.set('position', otherIdx);
```

Comment:
You can simply decrement and increment a single item's position to move it up or down I think

Reply:
Didn't you see if you click up or down button two rooms are swapping each other?
Although inserting action is modified creatively in my own way, I think this swapping action was reflected on issue requirements.

Reply:
I understand that. However, as complicated the solution, the more chances of bugs. As I have commented below, I am still getting an exception when I try to move items up or down

```
Uncaught TypeError: Cannot read property 'set' of undefined
    at n.moveRoom (open-event-frontend-8a9df458e3f4c6362f6d010561e81f79.js:982)
    at n.send (vendor-c31d19abb33ced0da35f2f8c34c17747.js:3328)
    at n [as send] (vendor-c31d19abb33ced0da35f2f8c34c17747.js:3279)
    at vendor-c31d19abb33ced0da35f2f8c34c17747.js:2097
    at a (vendor-c31d19abb33ced0da35f2f8c34c17747.js:3652)
    at vendor-c31d19abb33ced0da35f2f8c34c17747.js:2097
    at e.n._run (vendor-c31d19abb33ced0da35f2f8c34c17747.js:5193)
    at e.n._join (vendor-c31d19abb33ced0da35f2f8c34c17747.js:5192)
    at e.n.join (vendor-c31d19abb33ced0da35f2f8c34c17747.js:5157)
    at d (vendor-c31d19abb33ced0da35f2f8c34c17747.js:3735)
```

Reply:
As I mentioned in previous, old rooms should be removed at first.